static/sass/ds/6-components/_side-nav.scss

```diff
@@ -0,0 +1,39 @@
+.c-side-nav {
+  padding: $size-giant $size-xxl 0 $size-xxl;
+
+  ul {
+    list-style: none;
+  }
+
+  a {
+    color: $color-gray-dark;
+    transition: all .3s;
+
+    &:hover, &.is-active {
+      color: $color-gray-medium;
+    }
+  }
+
+  &__outer {
+    > li > a {
+      border-left-color: $color-white-off;
+      border-left-style: solid;
+      border-left-width: 3px;
+      color: $color-black-off;
+      display: block;
+      padding: $size-b 0 $size-b $size-b;
+
+      &:hover, &.is-active {
+        border-left-color: $color-yellow-tribune;
+      }
+    }
+  }
+
+  &__inner {
+    text-indent: 2.5rem;
+
+    > li {
+      margin-bottom: $size-b;
+    }
+  }
+}
```

Comment:
Could the `has-text-{color}` helpers do some work here?

Reply:
Yes! Done.